Cargo.toml

```diff
@@ -26,6 +26,7 @@ macros = ["hecs-macros", "lazy_static"]
 hecs-macros = { path = "macros", version = "0.3.0", optional = true }
 hashbrown = { version = "0.9.0", default-features = false, features = ["ahash", "inline-more"] }
 lazy_static = { version = "1.4.0", optional = true, features = ["spin_no_std"] }
+nohash-hasher = { version = "0.2.0" }
```

Comment:
```suggestion
nohash-hasher = { version = "0.2.0", default-features = false }
```
for no-std support

Reply:
Also, if you don't like pulling in this crate, it's easy to do this inline; here's where I tweaked legion's version of this: https://github.com/amethyst/legion/pull/204

Reply:
I don't have an objection in principle to a small dep, but reviewing nohash-hasher's impl I'm a bit confused by it: the type parameter seems to be completely meaningless. Am I missing something there? Considering the strangeness there, and the benefit of ensuring that the actual no-op hash implementation gets inlined, having our own impl might be better.